phpunit.xml.dist

```diff
@@ -3,7 +3,7 @@
          xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
          bootstrap="./vendor/autoload.php"
          colors="true"
-         executionOrder="random"
+         executionOrder="defects,random"
```

Comment:
this feature works in PHPUnit 10.5+, but only recently is allowed in phpunit.xml schema via https://github.com/sebastianbergmann/phpunit/pull/6258 which will be released with 10.5.48+

so while the feature works with older PHPUnit 10.5.x, tools which validate the xml - e.g. PHPStorm - would report that `defects,random` is not valid per schema.

Reply:
if I understand it correctly, Infection's config validation will not be broken as we validate a bit earlier, right?